index.js

```diff
@@ -96,8 +118,58 @@ const ScrollableTabView = React.createClass({
     }
   },
 
+  updateSceneKeys(children,sceneKeys=[],currentPage){
+    let newKeys=[]
+    children.forEach((child,idx) => {
```

Comment:
returned array is not a one to one mapping with children so map wouldnt serve the purpose here.
Also ill be checking for length of keys == #of children (#270)